lib/main.js

```diff
@@ -169,10 +178,17 @@ exports.main = function(options, callbacks) {
             Services.search.currentEngine = engine;
         }, 1000);
     }
+
     pageMod.PageMod({
       include: /^https?:\/\/(www|encrypted)\.google\..*\/.*$/,
       contentStyleFile: './css/nopopup.css',
     });
+
+    // only redirect if there is an atb parameter set
+    if (ss.storage.atb != undefined) {
```

Comment:
@bsstoner this makes sure that redirect happens only if there is an atb parameter set